app/src/main/java/com/github/fribourgsdp/radio/GameSettingsActivity.kt

```diff
@@ -1,11 +1,126 @@
 package com.github.fribourgsdp.radio
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.View
+import android.widget.*
+
+const val GAME_HOST_KEY = "com.github.fribourgsdp.radio.GAME_HOST"
```

Comment:
Here it would be possible to store them in the api.xml, since this class inherits from Activity, and therefore has a context. This was not possible in LyricsGetter.

Reply:
I looked up the kotlin documentation and they do it this way for intent keys.
What I was saying last time about your PR was about an api key given by the api creator.